docs/hugo/content/guide/frequently-asked-questions.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: FAQ
+weight: 0
+menu:
+  main:
+    weight: 0
```

Comment:
Be explicit about the weight here to ensure the ordering in the menu is the one we want. 
I think I've numbered existing top level items in increments of `10`.

Reply:
Just meant for this to be ordered in the sidebar not the menu. Have removed menu weight and left `weight`